### PR TITLE
Fixes #9601 - Adding a docker user guide

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: docs/user_guide/docker/docker.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+I can see here that my redis Repository has been added to a published Content View called redisv. If I want to use the tag from that Content View, I would just copy the Published At URL and then on my docker server I would run:
+
+```
+$ docker pull localhost:5000/default_organization-library-redisv-Tester-redis:latest
+Pulling repository localhost:5000/default_organization-library-redisv-Tester-redis...
+```
+
+## How to Provision Docker Content
+
+TBD.
```
Comment (daviddavis) [inline docs/user_guide/docker/docker.md]: Am waiting to see what foreman posts up in terms of documentation and tie this into their docs (with links, etc)

File: docs/user_guide/docker/docker.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Docker Management
```
Comment (jlsherrill) [inline docs/user_guide/docker/docker.md]: you're missing the header which causes it to be rendered on the menu

Comment (jlsherrill) [inline docs/user_guide/docker/docker.md]: in fact its not rendered at all, so i can't easily review

File: docs/user_guide/docker/docker.md (hunk 1 above)
Comment (jlsherrill) [inline docs/user_guide/docker/docker.md]: For now you might just link to the foreman docs?

Comment (jlsherrill) [inline docs/user_guide/docker/docker.md]: (the general foreman docs, not a particular section)

Comment (daviddavis) [inline docs/user_guide/docker/docker.md]: I added a link and some text based on what I guessed the foreman docs would contain. I'll revisit this later though once there's foreman-docker documentation.

File: docs/user_guide/docker/docker.md (hunk 2 above)
Comment (daviddavis) [inline docs/user_guide/docker/docker.md]: Done!

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**